validators/register-validator-node.md

```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
 
 <Step id='four'>
 
-## 4: Register Node with Network
+## 4: Setup SSL Certificates
 
-### Step 4.1: Register Node with Node CLI
+Here is an overview of the process to setup SSL certificates. For more details, please see [Node SSL docs](/validators/node-ssl.adoc).
 
-Note: Before proceeding, you will need to have at least  **0.2 Test ETH**. Also amount of delegated skale tokens need to be more or equal to minumum staking amount. Otherwise you will not be able to register with the SKALE Internal Devnet.
+### Step 4.1: Setup IP redirects
 
-To register with the network, you will need to provide the following:
+You will need to setup redirects from each node IP to the node domain.
 
-1.  Node name
-2.  Machine public IP
-3.  Port - beginning of the port range that will be used for skale schains (10000 recommended)
+### Step 4.2: Verify redirects
 
-**Terminal Command:**
+Verify that redirects work by sending a request to the watchdog API on 3009 port using your domain name.
 
-```bash
-skale node register --name [NODE_NAME] --ip [NODE_IP] --port [PORT]
+### Step 4.3: Issue SSL certificates
 
-```
-
-Optional arguments:
+You will need SSL certs issued by one of the Trusted CAs. Once you've decided on the certificate issuer you have several options - issue a separate certificate for each subdomain (node-0.awesome-validator.com, node-1.awesome-validator.com) or issue a single Wildcard SSL for all nodes (\*.awesome-validator.com). As a result, you should have 2 main files saved and copied to the respective nodes:
```

Comment:
Why 2?

Reply:
you mean the certificate and private key file?

Reply:
@ebruoguzberk yes, certificate and key